radix-engine-queries/src/typed_substate_layout.rs

```diff
@@ -123,6 +124,7 @@ pub enum TypedMainModuleSubstateKey {
     OneResourcePoolField(OneResourcePoolField),
     TwoResourcePoolField(TwoResourcePoolField),
     MultiResourcePoolField(MultiResourcePoolField),
+    TransactionTrackerField(TransactionTrackerField),
```

Comment:
We should also include the TransactionTrackerTrackedTransactionEntry here, otherwise we'll get errors when we try to ingest any user transactions.